src/app/sections/header/header.component.ts

```diff
@@ -1,15 +1,53 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit , EventEmitter, ViewChild } from '@angular/core';
+import { Router, NavigationEnd } from '@angular/router';
+import { Location } from "@angular/common";
 
 @Component({
   selector: 'header-section',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
+  providers: [Location]
 })
-export class HeaderSection implements OnInit {
+export class HeaderSection implements OnInit  {
 
-  constructor() { }
+  @ViewChild('HOME') home;
+  @ViewChild('SEND') send;
+  @ViewChild('ABOUT') about;
+  @ViewChild('CONTACT') contact;
 
-  ngOnInit() {
-  }
+  constructor (
+    private router: Router,
+    private location: Location) {
 
+    }
+
+    ngOnInit() {
+      this.router.events.subscribe ( event => {
+        if( event instanceof NavigationEnd ){
+          this.clickTab(this.getTabFromUrl(window.location.pathname))
+        }
+      });
+    }
+
+    getTabFromUrl(url){
+      let tab = url.split('/')[1]
+      return tab
+    }
+
+    clickTab(tab){
+      switch (tab) {
+        case '':
+          setTimeout(() => this.home.nativeElement.click(), 1)
```

Comment:
when doing hacks like this, id suggest setting the timeout to more than 1MS. It may not be enough time for some slow devices. Something like 10MS or even 100MS will not be noticed by the user, but will give the device plenty of time to reach this part of the code correctly.